src/Stylesheet.ts

```diff
@@ -21,11 +22,42 @@ export class Stylesheet {
                     return [match.groups?.className!, location] as const;
                 }),
         );
+
+        // To make sure we don't catch Sass variable declartions, we only look between :export {}
+        const variableExports = source.match(/:export\s*{\s*([^}]+)}/);
```

Comment:
I think you can simplify this with a better regular expression: `/(?<=:export\s*{[^}]*)(?<key>[^\s:}]+):\s*(?<value>.*?);(?=.*})/gs`

<img width="1765" alt="image" src="https://github.com/user-attachments/assets/520bc489-68d2-43fc-88b6-dcabab9c6496">

Playground [here](https://regex101.com/r/mVJ4CT/1)